rpc/src/v1/impls/devel.rs

```diff
@@ -93,4 +107,243 @@ where
+            TPSTestOption::TransferSingle => {
+                let mint_tx = mint_tx!(base_seq, 1);
+                let asset_type = asset_type(&mint_tx, shard_id);
+                let mut previous_tracker = mint_tx.tracker().unwrap();
+                send_tx(mint_tx, &*self.client, &genesis_keypair, &*self.miner)?;
+
+                let mut transactions = Vec::new();
+                for i in 0..count {
+                    let transfer_tx = transfer_tx!(
+                        base_seq + i + 1,
+                        vec![transfer_input!(previous_tracker, 0, asset_type, 1)],
+                        vec![transfer_output!(asset_type, 1)]
+                    );
+                    previous_tracker = transfer_tx.tracker().unwrap();
+                    let tx = sign_tx(transfer_tx, &genesis_keypair);
+                    transactions.push(tx);
+                }
+                transactions
+            }
+            TPSTestOption::TransferMultiple => {
+                let number_of_in_out: usize = 10;
+                let mint_tx = mint_tx!(base_seq, number_of_in_out as u64);
+                let asset_type = asset_type(&mint_tx, shard_id);
+                let mut previous_tracker = mint_tx.tracker().unwrap();
+                send_tx(mint_tx, &*self.client, &genesis_keypair, &*self.miner)?;
+
+                fn create_inputs(
+                    transaction_hash: H256,
+                    asset_type: H256,
+                    total_amount: u64,
+                    count: usize,
+                ) -> Vec<AssetTransferInput> {
+                    let mut inputs = Vec::new();
+                    let amount = total_amount / (count as u64);
+                    for i in 0..(count as usize) {
+                        let input = transfer_input!(transaction_hash, i, asset_type, amount);
+                        inputs.push(input);
+                    }
+                    inputs
+                }
+
+                let mut transactions = Vec::new();
+                for i in 0..count {
+                    let num_input = 1 + 9 * (i > 0) as usize;
+                    let inputs = create_inputs(previous_tracker, asset_type, number_of_in_out as u64, num_input);
+                    let outputs = vec![transfer_output!(asset_type, 1); number_of_in_out];
+
+                    let transfer_tx = transfer_tx!(base_seq + i + 1, inputs, outputs);
+                    previous_tracker = transfer_tx.tracker().unwrap();
+                    transactions.push(sign_tx(transfer_tx, &genesis_keypair));
+                }
+                transactions
+            }
+            TPSTestOption::PayOrTransfer => {
+                let mint_tx = mint_tx!(base_seq, 1);
+                let asset_type = asset_type(&mint_tx, shard_id);
+                let mut previous_tracker = mint_tx.tracker().unwrap();
+                send_tx(mint_tx, &*self.client, &genesis_keypair, &*self.miner)?;
+
+                let mut transactions = Vec::new();
```

Comment:
Same as the above.

Reply:
IMO, `with_capacity` looks more likely rust code.

Reply:
And there are no performance differences.

Reply:
Okay, I'll fix them.

Reply:
Fixed to `with_capacity()`.